backend/framework/qlf/dashboard/bokeh/qlf_plot.py

```diff
@@ -242,12 +242,19 @@ def mtable(qa, data, objtype=['OTYPE ?', 'OTYPE ?']):
             program_prefix = '_'+program
         else:
             program_prefix = ''
-        ref=par[key+program_prefix+'_REF']
-        if isinstance(ref,list):
+        try:
+            ref=par[key+program_prefix+'_REF']
+        except:
+            if qa =='xyshifts':
```

Comment:
seems like we will remove this code pretty soon

Reply:
I believe so..